template/partials/class.tmpl.partial

```diff
@@ -1,61 +1,62 @@
-    <li>The <code>address</code> properties of aggregate operators are automatically configured whereas a device operators requires manual configuration of its <code>address</code> property. This improves ease of using Open Ephys hardware.</li>
-    <li>All devices that belong to a hub (which is a collection of devices i.e. a headstage or breakout board) are automatically included, excluding the possibility of omitting configuration of a device that belongs to a hub. This improves ease of using Open Ephys hardware.</li>
-    <li>The workflow is less cluttered with configuration operators as one aggregate operator can correspond to multiple device operators. This improves workflow readability.</li>
+    <li>The <code>address</code> properties of aggregate configuration operators are automatically configured whereas a device configuration operators requires manual configuration of its <code>address</code> property. This improves ease of using Open Ephys hardware.</li>
+    <li>The workflow is less cluttered with configuration operators as one aggregate configuration operator can correspond to multiple device operators. This improves workflow legibility and expedites the workflow creation process.</li>
   </ul> 
 </div>
 
-{{/oe.device}}
+{{/oe.operator.configureDevice}}
+
+<div>
+  <h1 id="{{id}}" data-uid="{{oe.uid}}" class="text-break" style="display: inline-block;">
+    {{oe.name}}
+    {{#sourceurl}}<a class="header-action link-secondary" title="View source" href="{{sourceurl}}"><i class="bi bi-code-slash"></i></a>{{/sourceurl}}
+  </h1>
+  {{#oe.operator.type}}
+  <p style="display: inline; text-transform: capitalize"> 
+    {{oe.operator.type}} Operator
+  </p>
+  {{/oe.operator.type}}
+</div>
 
 <div>{{{oe.description}}}</div>
 
-{{#showWorkflow}}
-<div class="workflow"><p><img src="../workflows/operators/{{name.0.value}}.bonsai" alt="{{name.0.value}} Workflow"></p></div>
-{{/showWorkflow}}
+{{#oe.operator.workflow}}
 
-{{#oe.operatorType}}
+<div class="workflow"><p><img src="../workflows/operators/{{oe.name}}.bonsai" alt="{{oe.name}} Workflow"></p></div>
 
+{{#oe.operator}} 
 <h2>Inputs & Outputs</h2>
-
 {{>partials/diagram}}
+{{/oe.operator}}
 
-{{/oe.operatorType}}
-
-{{#oe.hasSubOperators}}
+{{/oe.operator.workflow}}
 
 <h2>Properties</h2>
 
-<div>{{name.0.value}} is a aggregate operator. It comprises of the following sub-operators: </div>
+{{#oe.operator.hub}}
+
+<p>{{oe.name}} is an aggregate operator. It comprises of the following sub-operators: </p>
 <br>
 
-{{#oe.subOperators}}
+{{#oe.properties}}
 
 <h3>{{{object}}}</h3>
 
-{{#subOperator}}
-<div>{{{object}}} is a {{{type}}} operator encapsulated by the {{name.0.value}} operator.</div>
-{{/subOperator}}
+{{#constituentOperator}}
+<p>{{{object}}} is a {{{type}}} operator encapsulated by the {{oe.name}} operator.</p>
+{{/constituentOperator}}
 
-{{#hasSubProperties}}
+{{^constituentOperator}}
+<p>These properties do not derive from a constituent operator.</p>
```

Comment:
Would it better to frame this as "These are properties of the headstage itself" instead of stating what they are not?